### PR TITLE
Final setup

### PR DESCRIPTION
**Pull request by Alejandro-Bernal-M**
# Math magicians: full website
    Full website ready

## Diferents pages using React Routes
A navigation bar was implemented, every link goes to a different page, and the URL changes using React Routes.
image

image

image

https://github.com/Alejandro-Bernal-M/Math.Magicians/blob/55cd823215e03edae2daddd3128599c3e8fa8474/src/App.js#L1-L21

## Styles
The styles were given using normal react styling and styles on modules.

https://github.com/Alejandro-Bernal-M/Math.Magicians/blob/55cd823215e03edae2daddd3128599c3e8fa8474/src/routes/Home.jsx#L1-L5

https://github.com/Alejandro-Bernal-M/Math.Magicians/blob/55cd823215e03edae2daddd3128599c3e8fa8474/src/App.js#L8

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,14 +1,21 @@
...
+import Home from './routes/Home';
+import CalculatorPage from './routes/CalculatorPage';
+import Quote from './routes/Quote';
+import NotMatch from './routes/NotMatch';
+import './styles/App.css';
+
+const App = () => (
+  <Routes>
+    <Route path="/" element={<Layout />}>
+      <Route index element={<Home />} />
```
Comment (tufoinnkuo10) [inline src/App.js:13]: - [optional] to make it more unique, you can highlight the navlinks so they indicate which page a user is currently in. great job. 👏🏾

File: src/components/apiCall.js [hunk 2]
```diff
@@ -46,6 +46,8 @@ function ApiCall() {
         &quot;
         {dataFromApi[0].quote}
         &quot;
+        -
+        {dataFromApi[0].author}
```
Comment (tufoinnkuo10) [inline src/components/apiCall.js:50]: - Great job on the Random quotes. 👏🏾 ✔️

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:✔️ ✔️ ✔️ 

Hi @Alejandro-Bernal-M ✔️ ✔️ ✔️ 

Your project is complete! Great job for nailing it right on the first attempt 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request✔️ 
- app works as expected✔️ 
- routes ok✔️ 
- pages ok ✔️ 
- No linter errors✔️ 

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

tufoin

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Alejandro-Bernal-M** [comment]
@tufoinnkuo10 Thanks for the review, I made the suggested changes.

**Pull request merged**